resources/authgear/templates/en/web/authflowv2/settings_profile_edit_birthdate.html

```diff
@@ -0,0 +1,40 @@
+{{ template "authflowv2/__html_head.html" . }}
```

Comment:
We don't need this. It should be included in `authflowv2/__settings_page_frame.html`